README.rst

```diff
@@ -25,106 +25,61 @@ Molecule
    :target: LICENSE
    :alt: Repository License
 
-Molecule is designed to aid in the development and testing of `Ansible`_ roles.
+Molecule is designed to aid in the development and testing of
+:std:doc:`Ansible <ansible:index>` roles.
```

Comment:
@decentral1se late notice: you cannot do this in README because it's included in PyPI.

* https://travis-ci.com/ansible/molecule/jobs/180975740#L238
* https://travis-ci.com/ansible/molecule/jobs/180975741#L238

Reply:
Fixed in `master`